subsys/testsuite/ztest/src/ztest_mock.c

```diff
@@ -245,6 +274,8 @@ int z_cleanup_mock(void)
 	int fail = 0;
 
 	if (parameter_list.next) {
+		printk("Not used: %s:%s\n", parameter_list.next->fn,
```

Comment:
Forgot to remove some debug code, but maybe it would be nice to make it prettier and leave it.